internal/query/current_sequence.go

```diff
@@ -194,7 +194,7 @@ func prepareLatestSequence() (sq.SelectBuilder, func(*sql.Row) (*LatestSequence,
 	return sq.Select(
 			CurrentSequenceColCurrentSequence.identifier(),
 			CurrentSequenceColTimestamp.identifier()).
-			From(currentSequencesTable.identifier() + " AS OF SYSTEM TIME '-1ms'").PlaceholderFormat(sq.Dollar),
+			From(currentSequencesTable.identifier()).PlaceholderFormat(sq.Dollar),
```

Comment:
can we discuss this together? i believe postgres won't support it, but we did this for crdb that we don't block the transactions